### PR TITLE
Request for code review

### PR DESCRIPTION
**Pull request by patriciachrysy**
Dear TSE,
i used the weather api for the weather infos and the gif API for the animated images. I separated the logic from the dom actions, give the temperatures in celsius and kelvin and made sure that the appearance changes according to the provision in the researched city.
Thanks for reviewing

### CODE REVIEW

File: src/api_handler.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
+  eslint-disable no-unused-vars, no-alert, no-mixed-operators, import/prefer-default-export,
+  no-empty
+*/
```
Comment (Oghenebrume50) [inline src/api_handler.js:3]: are all these warnings actually thrown by eslint or you just copied these into most of your files. For examples, I do not see why  "import/prefer-defautt-export" would come up when you are exporting multiple modules and not importing anything at all  :thinking:

File: src/api_handler.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      feels_like: {
+        celcius: (data.main.feels_like - 273.15).toFixed(1),
+        farenheit: ((data.main.feels_like - 273.15) * 9 / 5 + 32).toFixed(1),
+      },
+      temp_max: {
+        celcius: (data.main.temp_max - 273.15).toFixed(1),
+        farenheit: ((data.main.temp_max - 273.15) * 9 / 5 + 32).toFixed(1),
+      },
+    };
+  } catch (error) {
```
Comment (Oghenebrume50) [inline src/api_handler.js:36]: Why is the catch block empty, it should be doing something to handle any error that occurs here

File: src/app_logic.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    const left = leftSide(datas);
+    container.appendChild(left);
+
+    const image = await getGif(datas.description);
+    const rigth = rigthSide(image);
+    container.appendChild(rigth);
+
+    listenSearch();
+  } catch (error) {
+    alert(error);
```
Comment (Oghenebrume50) [inline src/app_logic.js]: Just alerting the actual error to the UI isn't really a good user experience, you do not expect your users to know how to read these errors

File: src/app_logic.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    const rigth = rigthSide();
+    container.appendChild(rigth);
+
+    listenSearch();
+  } catch (error) {
+    alert(error);
+  }
+};
+
+export { loadDefaultDatas };
```
Comment (Oghenebrume50) [inline src/app_logic.js]: it seems you are exporting just one function in this file, how about using export default and reduce the commented out warnings at the top

File: src/form_handler.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+const handleForm = (formId) => {
+  const form = document.getElementById(formId);
+  if (form.elements.namedItem('city').value) {
+    return form.elements.namedItem('city').value;
+  }
+  return null;
+};
+
+export { handleForm };
```
Comment (Oghenebrume50) [inline src/form_handler.js]: export default works here too

Also use the rules that are necessary in each file, all the commented out rules here do not look like an issue for this file at all, once you correct the export default

File: src/rigth_side_dom.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    const image = document.createElement('img');
+    image.src = './images/logo.gif';
+    image.alt = 'logo';
+    wrapper.appendChild(image);
+  }
+
+  return wrapper;
+};
+
+export { rigthSide };
```
Comment (Oghenebrume50) [inline src/rigth_side_dom.js]: export default

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi, @patriciachrysy good job working on this, you have done amazingly well, the UI looks great and the app works amazingly well. However, I would like to point out a few things, that should be changed and/or upgraded to work better. 

1. This shouldn't display just the error type, users cannot read this error, "city not found" is good enough or maybe changing the page to show a not found background, whatever works for you just do not dump the interpreter error back to the user.
error

2. You want to create a toggle for the temperature unit conversion, do not show both celsius and Fahrenheit at once.
see

Please make changes and request for another review, when done. 

**RAPHAEL NORIODE** :mask: STAY_SAFE

**Comment by patriciachrysy** [comment]
hey TSE,
applied the requested changes:
- added temperature unit button controls
- pop better errors
- corrected the imports and exports
- removed useless linter deactivation
Thanks for reviewing again

**Review by mosaaleb — APPROVED** [review]
#### Well done :clap:

You can merge :+1: 

##### Muhammad Ebeid

**Pull request merged**